cloud/pubnub_utils.js

```diff
@@ -0,0 +1,48 @@
+var PubNub = require('cloud/pubnub');
+
+module.exports.sendMessage = sendMessage;
+module.exports.replayChannel = replayChannel;
+
+// Returns a Promise. 
+// See https://parse.com/docs/cloudcode/guide#cloud-code-advanced
+function sendMessage(pubkey, subkey, channel, message, successCallback, errorCallback) {
```

Comment:
To keep the callback style consistent, instead of successCallback and errorCallback, just take an object response here that has success() and error() functions, similar to how our cloud code callbacks work